sormas-ui/src/main/java/de/symeda/sormas/ui/exposure/ExposureForm.java

```diff
@@ -69,8 +72,8 @@
 	private static final String LOC_BURIAL_DETAILS_HEADING = "locBurialDetailsHeading";
 
 	//@formatter:off
-	private static final String HTML_LAYOUT = 
-			fluidRowLocs(ExposureDto.UUID, ExposureDto.REPORTING_USER) +
+	private static final String UUID_REPORTING_USER = fluidRowLocs(ExposureDto.UUID, ExposureDto.REPORTING_USER);
```

Comment:
I don't really understand the need for this layout split. I'd prefer to just add a LOC to the layout and initialise/hide that based on the configuration criteria that you've used below.

Reply:
It's because of those conditions:
- show field only in for german instances
- show the survNet warning only for german systems and when the survNetGateway ist enabled.
I thought this looks a little better when those conditions are nested. And nested conditions in a string definition look ugly.

It's not really necessary to split the layout here, though. I will remove the split.

Reply:
I now remember: When the static strings are generated, the attribute epidataParentClass is not available. The epidataParentClass is necessary to determine how the HTML-Layout is supposed to look like, though (as the SurvNet-warning is only supposed to show for cases). The epiDataParentClass is usable in the non-static method createHtmlLayout().
As the layout is modified neither at the beginning, nor at the end, I decided to split the HTML_Layout to stuff before the modification part, and after. Taking this into consideration, I still don't see a better way to do that.

Reply:
Okay, fine by me then!